### ISSUE TITLE
OrderedDicts not defined even when DataStructures.jl is installed

### ISSUE THREAD

**Issue by kam3k** [JuliaLang/JSON.jl#94]
I have both JSON.jl and DataStructures.jl installed but cannot use `ordered=true` when calling `JSON.parsefile`.
```
julia> import JSON

julia> JSON.parsefile("test.json"; ordered=true, use_mmap=true)
ERROR: OrderedDict not defined
 in parse_object at /Users/marc/.julia/v0.3/JSON/src/Parser.jl:108
 in parse_value at /Users/marc/.julia/v0.3/JSON/src/Parser.jl:230
 in parse at /Users/marc/.julia/v0.3/JSON/src/Parser.jl:317
 in anonymous at /Users/marc/.julia/v0.3/JSON/src/JSON.jl:281
 in open at /Applications/Julia-0.3.5.app/Contents/Resources/julia/lib/julia/sys.dylib
 in parsefile at /Users/marc/.julia/v0.3/JSON/src/JSON.jl:279

julia> using DataStructures

julia> d = OrderedDict()
OrderedDict{Any,Any} with 0 entries
```
I am running Julia 0.3.5, DataStructures 0.3.5, and JSON 0.4.0 on OS X 10.10.2.

**Comment by kmsquire** [comment]
This was actually fixed by #92, but only on master.  I just tagged a new version, so `Pkg.update()` should fix this.  Sorry about that!

**Comment by kmsquire** [comment]
Try Pkg.update().  That problem was fixed
